pyproject.toml

```diff
@@ -74,6 +74,7 @@ bqstorage = [
 ]
 pandas = [
   "pandas >= 1.1.0",
+  "pandas-gbq >= 0.26.0",
```

Comment:
Update to 0.27.0 once https://github.com/googleapis/python-bigquery-pandas/pull/844 is merged and released

Reply:
Updated!